### DISCUSSION TITLE
"Name" a buffer without writing it to disk

### DISCUSSION THREAD

**Discussion by jerabaul29** [helix-editor/helix#9260] [Q&A] [answered]
Is there a way to "name" a buffer, without writing it to disk? For example, I can open a new buffer with ```:new``` which will be called ```[scratch]```; is there a way to change its name to ```[SOMETHING]``` (where there could be a convention like keeping the ```[]``` if it is not written to disk but just named?)?

**Comment by kirawi** [comment, marked as answer]
`:o <name>`

**Reply by jerabaul29** [reply]
Ahh, nice, many thanks :) I had not thought about this because I find the description a bit misleading (the documentation popup says "Open a file from disk into the current view.", but in this case there is no file yet). Many thanks!
